docs/start.md

```diff
@@ -36,8 +36,78 @@ See the [C API documentation](c/index.md).
 
 ## Python API
 
-Not yet written.
-
+This is a work in progress. To contribute to building Python bindings, in the project root:
```

Comment:
Can we move our changes here somewhere else (like a python-bindings.md, or as the README.md in the omf-python folder) since this is developer documentation rather than for API users.

Maybe put in a place holder link to the python docs, though it wont work currently.

Reply:
I’ve moved this text into `omf-python/README.md`.